src/DrDotnet/wwwroot/css/site.css

```diff
@@ -71,6 +71,8 @@ a, .btn-link {
 }
 .report code {
     color: #6200ee;
+    font-family: "Fira Code", monospace;
```

Comment:
Btw @Bluezen I used "Fira Code" font with a fallback to monospace, but I don't know if the font is embedded in drdotnet or downloaded from google fonts or if it works just because I happen to have it on my machine.